test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/Fixtures.html
+
+one:
+  email: MyString
+  password_digest: <%= BCrypt::Password.create('secret') %>
```

Comment:
Because the user model `has_secure_password`, you don't need to manually use BCrypt here to create the digest. You can simply set `password: "secret"` and all should be fine and dandy. :smiley: